plugins/Installation/lang/en.json

```diff
@@ -19,145 +19,145 @@
+        "NfsFilesystemWarningSuffixInstall": "Using file-based sessions on NFS is extremely slow, so Matomo will use database sessions. If you have many concurrent dashboard users, you may need to increase the maximum number of client connections to the database server.",
+        "NoConfigFileFound": "Could not find the Matomo configuration file, and you are trying to access a Matomo page.",
         "YouMayInstallPiwikNow": "You may %1$sinstall Matomo now%2$s",
-        "IfPiwikInstalledBeforeTablesCanBeKept": "If you installed Matomo before and have some tables in your DB, don't worry, you can reuse the same tables and keep your existing data!",
+        "IfPiwikInstalledBeforeTablesCanBeKept": "If you installed Matomo before and have some tables in your database, don't worry, you can reuse the same tables and keep your existing data.",
         "Optional": "Optional",
         "Password": "Password",
-        "PasswordDoNotMatch": "password do not match",
+        "PasswordDoNotMatch": "mismatching passwords",
         "PasswordRepeat": "Password (repeat)",
         "PercentDone": "%s %% Done",
-        "ProfessionalServicesAdTitle": "Advanced Analytics & Services",
-        "ProfessionalServicesfessionalServicesAdText": "Our network of experts can help your organization unlock the full power of Matomo analytics.",
+        "ProfessionalServicesAdTitle": "Advanced Analytics and Services",
+        "ProfessionalServicesfessionalServicesAdText": "Our network of experts can help your organization use Matomo analytics to its full potential.",
         "ProfessionalServicesfessionalDiscoverHow": "Learn about %1$sadditional products and services%2$s for Matomo and how to increase your conversions and revenue.",
         "PleaseFixTheFollowingErrors": "Please fix the following errors",
         "DefaultSettings": "Default Matomo settings",
         "DefaultSettingsHelp": "Matomo comes with default settings. You can customize them now or do so later in the admin screen.",
         "Requirements": "Matomo Requirements",
         "RestartWebServer": "After making this change, restart your web server.",
         "ReusingTables": "Reusing the Tables",
-        "PiwikOrgNewsletter": "email me with major Matomo community updates",
-        "ProfessionalServicesNewsletter": "send me information on %1$sProfessional services and products%2$s for Matomo",
-        "SeeBelowForMoreInfo": "See below for more information.",
-        "SetupWebsite": "Setup a Website",
-        "SetupWebsiteError": "There was an error when adding the website",
+        "PiwikOrgNewsletter": "email me about major Matomo community updates",
+        "ProfessionalServicesNewsletter": "send me info on %1$sProfessional services and products%2$s for Matomo",
+        "SeeBelowForMoreInfo": "More info available below.",
+        "SetupWebsite": "Set up a Website",
+        "SetupWebsiteError": "Could not add the website",
```

Comment:
Maybe revert this one for the more clear existing one.

Reply:
Forgot to say I am guessing here. Sorry about that.
Is the website added or not? What is the error?